src/content/mod_configuration.cpp

```diff
@@ -226,6 +227,18 @@ void ModConfiguration::resolveDependencies()
 		modnames.insert(mod.name);
 	}
 
+	// Step 1.5 (optional): shuffle unsatisfied mods so non declared depends get found by their devs
+	if (g_settings->getBool("random_mod_load_order")) {
+	    std::random_device rd;
+    	std::mt19937 g(rd());
+		//std::random_shuffle would be better here, but it has been removed in C++17
```

Comment:
I don't see how

Reply:
Well, I wouldn't have had to pass the generator